tests/torchtune/datasets/test_concat_dataset.py

```diff
@@ -90,3 +90,33 @@ def test_packed_dataset(self, torch_datasets):
 
         with pytest.raises(ValueError):
             concated_dataset = ConcatDataset(torch_datasets)
+
+    def test_all_packed_datasets(self, torch_datasets):
```

Comment:
should also test error is caught when some datasets are unpacked?

Reply:
Oh yeah it already exists in the previous unit test